solutions/7.js

```diff
@@ -0,0 +1,10 @@
+const solution = (num, i = 1, bigD = -1) => {
+  if (i < num) {
+    if (num % i == 0 && bigD < i) {
```

Comment:
No matter what, `i` will always be bigger than `bigD`. Therefore, you can just remove `bigD < i` logic and assign value `i` to `bigD`.